### PR TITLE
Fix template name format 

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

Using the right Twig template paths according to https://symfony.com/doc/5.4/templates.html#bundle-templates

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Everything must work as before.

### CODE REVIEW

**Comment by mabumusa1** [comment]
I wrote this script to find and replace twig paths to match default symfony format 

```
<?php
class finder {
  /**
     * @param string $directory
     *
     * @return SplFileInfo[]
     */
    public function getAllFiles(string $directory = './'): array
    {
        $result = [];

        $iterator = new RecursiveIteratorIterator(new RecursiveDirectoryIterator($directory), RecursiveIteratorIterator::SELF_FIRST );
        /** @var $fileInfo SplFileInfo */
        foreach ($iterator as $fileInfo ) {
            if ($fileInfo->isFile()) {
                if (!$fileContent = file_get_contents($fileInfo->getPathName())){
                    echo "Error reading file: " . $fileInfo->getPathName() . "\n";
                }
                if(preg_match_all("/'[^']*:[^']*:[^']*.twig'/", $fileContent, $keys, PREG_PATTERN_ORDER)){
                    foreach($keys[0] as $key){
                        $replace = str_replace("Bundle", "", $key);
                        $replace = str_replace("Mautic", "@Mautic", $replace);
                        $replace = str_replace(":", "/", $replace);
                        
                        $content = str_replace ($key, $replace, $fileContent);
                        
                        if (!file_put_contents ($fileInfo->getPathName(), $content)) {
                            echo "Error writing file: " . $fileInfo->getPathName() . "\n";
                            $result[] = $fileInfo->getPathName();
                        }
                    }                    
                }
            }
        }

        return $result;
    }
}

$x = new finder();
var_dump($x->getAllFiles());

```

**Review by escopecz — CHANGES_REQUESTED** [review]
It seems that some tests are broken by this change. Please take a look.

**Comment by mabumusa1** [comment]
> It seems that some tests are broken by this change. Please take a look.

Yes, the form scripts is the one causing the issue. I solved it waiting for your confirmation over slack

**Comment by mabumusa1** [comment]
This issue should be resolved as part of the templating refactoring

https://github.com/mautic/mautic/issues/12107

File: app/bundles/FormBundle/Model/FormModel.php
Comment (escopecz) [inline app/bundles/FormBundle/Model/FormModel.php:788]: @mabumusa1 and I discussed via PM that he will take care of this todo when removing the TemplatingHelper

**Review by escopecz — APPROVED** [review]
Thank you! Since this was an automated replacement PR and all tests are passing then I'm merging this to unblock the removal of the PHP templating engine.

**Pull request merged**
